lnrpc/rpc.swagger.json

```diff
@@ -3420,6 +3482,25 @@
           "title": "/ The unconfirmed balance of a wallet(with 0 confirmations)"
         }
       }
+    },
+    "lnrpcWitnessType": {
+      "type": "string",
+      "enum": [
+        "COMMITMENT_TIME_LOCK",
+        "COMMITMENT_NO_DELAY",
+        "COMMITMENT_REVOKE",
+        "HTLC_OFFERED_REVOKE",
+        "HTLC_ACCEPTED_REVOKE",
+        "HTLC_OFFERED_TIMEOUT_SECOND_LEVEL",
+        "HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL",
+        "HTLC_OFFERED_REMOTE_TIMEOUT",
+        "HTLC_ACCEPTED_REMOTE_SUCCESS",
+        "HTLC_SECOND_LEVEL_REVOKE",
+        "WITNESS_KEY_HASH",
+        "NESTED_WITNESS_KEY_HASH"
+      ],
+      "default": "COMMITMENT_TIME_LOCK",
```

Comment:
perhaps we should also define an UNKNOWN_WITNESS type so that it can be the default.

Reply:
Done.